docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nvidia/cuda:11.7.1-runtime-ubuntu20.04
+
+RUN apt-get update -y && apt-get install -y python3 python3-pip 
+RUN pip install fschat
```

Comment:
Would it not be better to copy source files inside of the image and installed the package from local code?

```
pip install -e .
```

Reply:
Indeed, this was a copy and paste from use in another project of mine.